bin/rucio-admin

```diff
@@ -1192,9 +1205,9 @@ Commands:
     # The declare-bad command
     declare_bad_file_replicas_parser = rep_subparser.add_parser('declare-bad', help='Declare bad file replicas')
     declare_bad_file_replicas_parser.set_defaults(which='declare_bad_file_replicas')
-    declare_bad_file_replicas_parser.add_argument(dest='listbadfiles', action='store', nargs='*', help='The list of bad files')
+    declare_bad_file_replicas_parser.add_argument(dest='listbadfiles', action='store', nargs='*', help='List of bad file PFNs (for one replica) or LFNs (for all replicas)')
     declare_bad_file_replicas_parser.add_argument('--reason', dest='reason', required=True, action='store', help='Reason')
-    declare_bad_file_replicas_parser.add_argument('--inputfile', dest='inputfile', nargs='?', action='store', help='Reason')
+    declare_bad_file_replicas_parser.add_argument('--inputfile', dest='inputfile', nargs='?', action='store', help='File containing list of bad files')
 
```

Comment:
from the discussion today, maybe something like "--allow-collection" to enable marking of datasets and containers

Reply:
@cserf opinion?

Reply:
Yes. I think a simple check if the input did is a FILE or DATASET/CONTAINER. If dataset/container the --allow-collection has to be used.

Reply:
Implemented --allow-collection safeguard.
One last point: maybe the list of PFNs that will be declared bad should be printed out? Particularly if it's generated from a DID

Reply:
I think a print out would be good yes.

Reply:
done!